src/api/app/views/layouts/webui/_login_modal.html.haml

```diff
@@ -2,7 +2,7 @@
   .modal-dialog.modal-dialog-centered{ role: 'document' }
     .modal-content
       .modal-body
-        %i.close.fas.fa-times{ data: { dismiss: 'modal' } }
+        %i.btn.btn-close.float-end{ type: 'button', data: { dismiss: 'modal' } }
```

Comment:
```suggestion
        %button.btn.btn-close.float-end{ type: 'button', data: { dismiss: 'modal' } }
```
And lastly, we could use the `button` element instead of the `i` element, as described in the docs...